docs/release-notes.txt

```diff
@@ -3,6 +3,134 @@
 Release Notes
 =============
 
+Version 0.13
+------------
+
+Version 0.13.0
+^^^^^^^^^^^^^^
+
+**Release Date:** August 20, 2020
+
+**New Features**
+
+- Allow experiments to connect to the master using TLS.
+
+- Allow agent's TLS verification to skip verification or use a custom certificate for the master.
+
+- For ``TFKerasTrial`` and ``EstimatorTrial``, add functionality to ``context.wrap_dataset()`` which allows users to skip sharding their training dataset for distributed training. If using this option, users are responsible for splitting their dataset in such a manner that every GPU (rank) sees unique data.
+
+**Improvements**
+
+- **Breaking Change:** Change how additional arguments to ``ssh`` are passed through ``det shell start`` and ``det shell open``.  Previously they were passed as a single string, like ``det shell open SHELL_ID --ssh-opt '-X -Y -o SomeSetting="some string"'``, but now the ``--ssh-opt`` has been removed and all extra positional arguments are passed through without requiring double-layers of quoting, like ``det shell open SHELL_ID -- -X -Y -o SomeSetting="some string"`` (note the use of ``--`` to indicate all following arguments are positional arguments).
+
+- Install the ``determined`` and ``determined_cli`` Python packages inside ``det shell``.  Also, pass any user-defined environment variables for the task image into the ssh sessions opened via ``det shell start`` or ``det shell open``.
+
+- **Remove Determined Steps from the UX:** Remove the concept of a "Determined step" from the UX and add new configuration settings to allow settings previously in terms of steps to be configured instead in terms of records, batches or epochs. See the :ref:`migration guide <migration-guide_remove-steps>` for details on migrating from the old configuration to the new configuration.
+
+  - Many configuration settings can now be set in terms of records, batches or epochs. For example, a single searcher can be configured to run for 100 records by setting ``max_length: {records: 100}``, 100 batches by setting ``max_length: {batches: 100}``, or 100 epochs by setting ``records_per_epoch`` at the root of the config and ``max_length: {epochs: 100}``.
+  - A new configuration setting, ``records_per_epoch``, is added that must be specified when any quantity is configured in terms of epochs.
```

Comment:
Here and below, let's make the tense consistent -- we don't normally write in past tenses for changes in the current release ("was replaced" -> "has been replaced" or "is replaced by")